.circleci/config.yml

```diff
@@ -69,14 +69,15 @@ jobs:
           name: TorchBench run
           command: |
             source .circleci/setup_env.sh
-            python benchmarks/torchbench.py --ci --repeat 2 --coverage -d cuda --raise-on-assertion-error --raise-on-backend-error -x Super_SloMo -x moco
+            python benchmarks/torchbench.py --skip-fp64-check --coverage -d cuda
```

Comment:
hmm @desertfire I'm not sure why your changes are showing up in my PR - it looks like https://github.com/pytorch/torchdynamo/pull/1040/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47 already landed.

Reply:
Update main and then `git rebase`?